docs/docker.md

```diff
@@ -8,13 +8,134 @@ Start by downloading and installing Docker CE for your platform:
 * [Windows](https://docs.docker.com/docker-for-windows/install/)
 * [Linux](https://docs.docker.com/install/)
 
+Optionally, [docker-compose](https://docs.docker.com/compose/install/) should be installed and available to follow the docker quick start guide.
+
 Once you have Docker installed, simply prepare the config file (e.g. `config.json`) and run the image for `freqtrade` as explained below.
 
-## Download the official FreqTrade docker image
+## Freqtrade with docker-compose
+
+Freqtrade provides an official Docker image on [Dockerhub](https://hub.docker.com/r/freqtradeorg/freqtrade/), as well as a [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) ready for usage.
+
+!!! Note
+    The following section assumes that docker and docker-compose is installed and available to the logged in user.
+
+!!! Note
+    All below comands use relative directories and will have to be executed from the directory containing the `docker-compose.yml` file.
+
+### Docker quick start
+
+Create a new directory and place the [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) in this directory.
+
+``` bash
+mkdir freqtrade
+cd freqtrade/
+# Download the docker-compose file from the repository
+curl https://raw.githubusercontent.com/freqtrade/freqtrade/develop/docker-compose.yml -o docker-compose.yml
+
+# Pull the freqtrade image
+docker-compose pull
+
+# Create user directory structure
+docker-compose run --rm freqtrade create-userdir --userdir user_data
+
+# Create configuration - Requires answering interactive questions
+docker-compose run --rm freqtrade new-config --config user_data/config.json
+```
+
+The above snippet will create a directory called "freqtrade" - download the latest compose file and pull the freqtrade image.
+The last 2 steps will create the user-directory, as well as a default configuration based on your selections.
+
+#### Adding your strategy
+
+The configuration is now available as `user_data/config.json`.
+You should now copy your strategy to `user_data/strategies/` - and add the Strategy class name to the `docker-compose.yml` file, replacing `SampleStrategy`.
+
+Once this is done, you're ready to launch the bot in trading mode.
```

Comment:
```suggestion
Once this is done, you're ready to launch the bot in trading mode (Dry-run or Live-trade, depending on your answer to the corresponding question you made above).
```

Reply:
Somewhere a sentence should be placed, that the user can edit config, changing settings in it and what actions should be done after it (or no additional actions).